be/src/runtime/load_channel.cpp

```diff
@@ -142,7 +142,7 @@ void LoadChannel::open(brpc::Controller* cntl, const PTabletWriterOpenRequest& r
                     st = local_tablets_channel->incremental_open(request, _schema);
                 }
             } else {
-                st = Status::NotSupported("incremental open not supported by this tablets channel");
+                st = it->second->incremental_open(request, _schema);
```

Comment:
there is concurrent issue between write and incremental open?

Reply:
done